hedera-node/hedera-consensus-service-impl/src/test/java/com/hedera/node/app/service/consensus/impl/test/handlers/ConsensusGetTopicInfoHandlerTest.java

```diff
@@ -139,10 +145,28 @@ void validatesQueryIfInvalidTopic() throws Throwable {
     }
 
     @Test
+    @DisplayName("Topic Id in transaction is needed during validate")
```

Comment:
This display name is much more useful! 👍

Reply:
thanks.